lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -506,6 +508,13 @@ void sgw_s8_handle_create_session_response(
     sgw_remove_sgw_bearer_context_information(
         sgw_state, session_rsp_p->context_teid, imsi64);
   }
+
+  char imsi_str[IMSI_BCD_DIGITS_MAX + 1];
+  IMSI64_TO_STRING(imsi64, (char*) imsi_str, IMSI_BCD_DIGITS_MAX);
+  char teidString[16];
+  sprintf(teidString, "%u", session_rsp_p->context_teid);
+  directoryd_update_field(imsi_str, "sgw_c_teid", teidString);
```

Comment:
Need to update only for success case.

Reply:
yep. I put it here as an example. Not sure where to put it, either s8_task or mme_task

What we need is to get all the available Teids and put them in a comma separated string. So directoryd record would be like 

```
{
  "identifiers": {
    "sgw_c_teid": "1467, 3688, 31093"
  },
  "location_history": [
    "d35eb697-ad83-4dee-a049-6e15273dd0ee"
  ],
  "py/object": "magma.directoryd.rpc_servicer.DirectoryRecord"
}
```

where `sgw_c_teid` is all the default bearer teids. for that subscriber

Reply:
for now we can keep it here so we can test it in teravm using FEG. 
Once it is landed we can move around this statement and place it in a proper place